ci/Jenkinsfile.linux

```diff
@@ -82,7 +82,7 @@ pipeline {
       steps { script {
         /* Temporary hack-fix for failing Linux builds:
          * https://github.com/status-im/infra-ci/issues/88 */
-        timeout(10) { retry(20) {
+        timeout(10) { retry(10) {
```

Comment:
This is lowering it from 20 to 10, is that intentional? Seems wrong. And commit/pr title is wrong too.

Reply:
I cherry-picked your commit here to have it in the release branch: https://github.com/status-im/status-desktop/pull/11248/commits/a5ba25373091d2472759f07fb61041d317dee6d0

I don't mind dropping this PR if it's not gonna help

Reply:
It will only make things worse, since less retries means less opportunities to hit a successful build.